docs/user/dashboard/vega.asciidoc

```diff
@@ -1,82 +1,7 @@
-[[tutorials]]
-== Tutorials
+[[vega]]
+=== Add custom panels with Vega
 
-Learn how to use *Lens*, *Vega*, and *Timelion* by going through one of the step-by-step tutorials.
```

Comment:
Now that you've combined the Vega-Lite and Vega tutorials into one page, I think we need an opening sentence that explains that it's two separate tutorials.

`Vega panels have two modes. The first tutorial is for the simpler Vega-Lite visualizations, followed by a separate tutorial for making more complex and interactive Vega visualizations.`